### ISSUE TITLE
[DOCS] - Documentation/Guide relocation

### ISSUE THREAD

**Issue by theredkoala21** [Ylianst/MeshCentral#233]
I had a re read over the docs recently. I found it hard to follow and find something I was looking for. As it was multiple PDF documents. (Installer guide and user guide)

What if we moved over to a web based docs system?

Something like:
https://readthedocs.org/

**Comment by Ylianst** [comment]
Interesting, I did not know about this site. Sort of busy right now with many issues so, not high priority. But this would be nice.

**Comment by Ylianst** [comment]
Would be nice to do, but I am swamped with work. Not a high priority right now, but good suggestion.

**Comment by MailYouLater** [comment]
Just to update this issue with the bits of conversation on this topic that have appeared in other issues since the last comment in this issue:

Back in October 2019, we had a brief discussion about moving the docs to somewhere the community could help maintain them on Issue #579 which ended with @Ylianst adding a ["docs" folder](https://github.com/Ylianst/MeshCentral/tree/master/docs) to the repository which contains the current documentation in MS Word format, and with the following comments:
> [Ylianst said](https://github.com/Ylianst/MeshCentral/issues/579#issuecomment-544072132): If someone has time to prototype taking one of the smallest documents and place it on Read The Docs. I would love to see what it would look like. Especially the pictures, etc. I am a bit over worked these days, I need to see if I team can get me a tech writer at some point.

> [MailYouLater said](https://github.com/Ylianst/MeshCentral/issues/579#issuecomment-544110140): Which document would you recommend someone think about converting for a smaller, sample Read the Docs project?

Just a couple of days ago (January 2020) this idea was [mentioned](https://github.com/Ylianst/MeshCentral/issues/830#issuecomment-575859616) again in issue #830, where I then suggested we centralize discussion of this in the existing issue (this one).

**Comment by Jorgen-P** [comment]
If the plan is to relocate to readthedocs which uses markdown or RestructuredText (prefered), converting the documentation to RestructuredText (which Github can render so at least the text would look ok living on GH for now, not sure about images) seems like a reasonable next step. The docs use a lot of images so getting those working will probably be necessary for a conversion.

I tried converting an odt file to rst using pandoc but it had trouble picking up images (but worked fine on a test odt I made), I suspect this has to do with the images inside the odt file being located in "media" instead of "Pictures", probably as a result of being converted. Pandoc did manage to get images from the old .docx documentation. If the images have not changed much since the docx->odt conversion (which seems to be from early May) it might work to convert both and merge them as plain text to get the images from the older documentation. In cases where the images have been updated manually unzipping the odt file to get at the images does work as a last resort.
